app/models/hackathon/swag_request/delivered.rb

```diff
@@ -0,0 +1,22 @@
+module Hackathon::SwagRequest::Delivered
+  extend ActiveSupport::Concern
+  extend Suppressible
+
+  def deliver_later_if_pertinent(wait: nil)
+    Hackathons::SwagRequestDeliveryJob.set(wait:).perform_later(self) if pertinent? && !suppressed?
+  end
+
+  def deliver_if_pertinent
+    HackathonMailer.with(hackathon:).swag_request.deliver if pertinent? && !suppressed?
```

Comment:
are we touching `delivered_at` anywhere?

Reply:
Ah, good catch! I'll touch it here.